src/main/java/com/epam/test_generator/entities/Token.java

```diff
@@ -1,21 +1,22 @@
 package com.epam.test_generator.entities;
 
+import com.epam.test_generator.entities.api.TokenTrait;
 
 import java.util.Calendar;
 import java.util.Date;
+import java.util.UUID;
 import javax.persistence.Entity;
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
 import javax.persistence.OneToOne;
 import javax.validation.constraints.NotNull;
 
-
 /**
  * This class represents token essence. Token is a special key that is used for user identification.
  */
 @Entity
-public class Token {
+public class Token implements TokenTrait {
```

Comment:
Can you please fix javadoc by renaming "essence" to "entity", and adding cases what token is actually used for.

Reply:
Fixed